src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -75,6 +75,29 @@ customize the `LogRecordExportProcessorOptions` or see the [Environment
 Variables](#environment-variables) section below on how to customize using
 environment variables.
 
+The example has a
```

Comment:
IIRC this is copied from some existing place, @vishweshbankwar could you reduce the duplication by restructuring the docs? Ideally, we don't want to keep copying stuff.

Reply:
@reyang - I pushed an update. Is this better?